test/functional/test_framework/test_node.py

```diff
@@ -140,6 +142,29 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, bitcoind, bitcoin_cl
             AddressKeyPair('mzRe8QZMfGi58KyWCse2exxEFry2sfF2Y7', 'cPiRWE8KMjTRxH1MWkPerhfoHFn5iHPWVK5aPqjW8NxmdwenFinJ'),
     ]
 
+    def is_valgrind_present(self):
+        """Check for valgrind presence"""
+        try:
+          version = subprocess.run(["valgrind", "--version"], stdout=subprocess.PIPE).stdout.decode("utf-8")
+          if version.find("valgrind", 0, 10) > -1:
```

Comment:
`if version.startswith("valgrind-")` :)

Reply:
Believe it or not, I was just typing the same changes. Thanks!